PAT/javaSrc/PAT-B1004.java

```diff
@@ -0,0 +1,41 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class B1004 {
+    //定义student类
```

Comment:
```suggestion
    // 定义student类
```
规范化注释：// 后面加一个空格 " "